browser/ui/brave_browser_command_controller.cc

```diff
@@ -366,9 +367,9 @@ void BraveBrowserCommandController::UpdateCommandForBraveVPN() {
 void BraveBrowserCommandController::UpdateCommandForPlaylist() {
 #if BUILDFLAG(ENABLE_PLAYLIST_WEBUI)
   if (base::FeatureList::IsEnabled(playlist::features::kPlaylist)) {
-    UpdateCommandEnabled(
-        IDC_SHOW_PLAYLIST_BUBBLE,
-        browser_->is_type_normal() && !browser_->profile()->IsOffTheRecord());
```

Comment:
I think it'd be safe to leave `browser_->is_type_noraml()`, so that the command is disabled in popup or other types of windows.